vcap/vcap/batch_executor.py

```diff
@@ -57,33 +58,41 @@ def __init__(self, batch_fn: Callable[[List[Any]], List[Any]],
     def total_imgs_in_pipeline(self) -> int:
         return self._request_queue.qsize() + self._num_imgs_being_processed
 
-    def submit(self, input_data: Any, output_queue: Queue = None) -> Queue:
-        """Creates an OvenRequest for you and returns the output queue"""
-        output_queue = output_queue if output_queue else Queue()
-        self._request_queue.put(_OvenRequest(
-            output_queue=output_queue,
+    def submit(self, input_data: Any, future: Future = None) -> Future:
+        """Submits a job and returns a Future that will be fulfilled later."""
+        future = future or Future()
+
+        # Add backwards compatibility for 0.2
+        future.get = future.result
```

Comment:
This monkeypatching has worked such that the BrainFrame platform and all capsules on the aotu.ai library work perfectly, so I think it's a good enough backwards-compatibility fix for users of the API, since it's still young in userbase.